src/app/people/people.component.ts

```diff
@@ -19,7 +19,9 @@ export class PeopleComponent implements OnInit {
   constructor(private HttpService: PeopleService) { }
 
   ngOnInit() {
-    this.people = this.HttpService.getPeople().map(data => data.results);
-    this.people.subscribe(() => this.showSpinner = false);
+    this.people = this.HttpService.getPeople().map(data => {
```

Comment:
If you declare `Observable` variable, please add `$` at the end of variable name

Reply:
Move `map` to your service, don't modify data in component, just invoke `getPeople` method

Reply:
Done